src/prefabs/form.tsx

```diff
@@ -63,11 +63,11 @@ const beforeCreate = ({
     'ZIPCODE',
   ];
 
-  const [modelId, setModelId] = React.useState(null);
-  const [model, setModel] = React.useState(null);
+  const [modelId, setModelId] = React.useState('');
+  const [model, setModel] = React.useState('');
```

Comment:
Shouldn't this be an empty object?
And why can't we keep it null?

Reply:
otherwise the compilation will fail, namely this mutation `validatePageArtifactMutation`

Reply:
We can also fix that in the backend, by allowing null.